src/views/Run/internal/ResultDisplay/internal/ReportSection/internal/ReportSection.helpers.ts

```diff
@@ -0,0 +1,13 @@
+import { RunReport } from '@/benchttp'
+
+export const getFailedTestCount = (
+  results: RunReport['tests']['results']
+): number => {
+  return results.reduce<number>((acc, curr) => {
+    if (!curr.pass) {
+      acc = acc++
```

Comment:
```suggestion
      acc = ++acc
```

Reply:
My bad `acc++` increments the variable with no need of manually re-assigning. Does  `acc = ++acc` seems cleaner to you than `acc++` ?

Reply:
I thought it would work too but it didn't when I tested it, it was always equal to 0.
Both `acc = ++acc` and `acc += 1` work though.
Maybe because it adds +1 after the assignation when ++ is at the end?

Reply:
I'd go for `acc += 1` and call it a day. This way we don't have to deal with thinking about pre or postfix notation details.